tests/ci_tests/code_formatter/crewai/test_crewai_tools.py

```diff
@@ -0,0 +1,182 @@
+"""Tests for coffee_maker/code_formatter/crewai/tools.py"""
+
+from unittest import mock
+
+import pytest
+from pydantic import ValidationError
+
+from github import GithubException
+
```

Comment:
Imports have been regrouped and sorted alphabetically according to the style guide. Standard library, third-party, and local application imports are now in separate, alphabetized groups, each separated by a blank line for clarity.
```suggestion
from unittest import mock

from github import GithubException
from pydantic import ValidationError
import pytest

from coffee_maker.code_formatter.crewai.tools import PostSuggestionInput, PostSuggestionToolLangAI
```